pyro/distributions/transforms/permute.py

```diff
@@ -83,3 +84,21 @@ def log_abs_det_jacobian(self, x, y):
         """
 
         return torch.zeros(x.size()[:-1], dtype=x.dtype, layout=x.layout, device=x.device)
+
+
+def permute(input_dim, permutation=None):
+    """
+    A helper function to create a :class:`~pyro.distributions.transforms.Permute` object for consistency with other
+    helpers.
+
+    :param input_dim: Dimension of input variable
+    :type input_dim: int
+    :param permutation: Torch tensor of integer indices representing permutation. Defaults
+        to a random permutation.
+    :type permutation: torch.LongTensor
+
+    """
+
+    if permutation is None:
+        permutation = torch.randperm(input_dim, device='cpu').to(torch.Tensor().device)
```

Comment:
optimal device usage? @fritzo ?

Reply:
`torch.randperm` now has cuda support. Please change to simply:
```py
permutation = torch.randperm(input_dim)
```

Reply:
Done!